### PR TITLE
Storybook: Fix control types

### PR DESCRIPTION
**Pull request by tyxla**
## What?
This fixes control types so they work both in Storybook 7 and 8.

Part of #67574

## Why?
Preparatory work for upgrading Storybook v8 - `false` controls cause type errors with v8.

## How?
Just updating the types.

## Testing Instructions
Verify Storybook builds well and test a few controls to ensure they work the same way.

### Testing Instructions for Keyboard
Same

## Screenshots or screencast 
None

### CODE REVIEW

File: packages/block-editor/src/components/alignment-control/stories/aliginment-toolbar.story.js [hunk 1]
```diff
@@ -16,13 +16,13 @@ const meta = {
 	component: AlignmentToolbar,
 	argTypes: {
 		value: {
-			control: { type: false },
+			control: { type: undefined },
```
Comment (mirka) [inline packages/block-editor/src/components/alignment-control/stories/aliginment-toolbar.story.js]: I think we misread the [docs](https://storybook.js.org/docs/api/arg-types#control) 😂 Does this pass?

```suggestion
			control: false,
```

Comment (tyxla) [reply packages/block-editor/src/components/alignment-control/stories/aliginment-toolbar.story.js]: Lol, looks like it does. Updated in https://github.com/WordPress/gutenberg/pull/67646/commits/e2b13c08106c8fd71fdebc278585460222e2e301

**Review by mirka — APPROVED** [review]
Thanks 🙈

**Comment by tyxla** [comment]
Thanks Lena 🙇

**Pull request merged**